gems/pending/appliance_console/database_replication_master.rb

```diff
@@ -0,0 +1,46 @@
+require 'appliance_console/logging'
+require 'appliance_console/prompts'
+require 'appliance_console/database_configuration'
+require 'appliance_console/database_replication'
+
+module ApplianceConsole
+  class DatabaseReplicationMaster < DatabaseReplication
+    include ApplianceConsole::Logging
+
+    REGISTER_CMD = 'repmgr master register'.freeze
+
+    def initialize(hash = {})
+      set_defaults
+      super
+    end
+
+    def set_defaults
+      self.host         = LinuxAdmin::NetworkInterface.new(NETWORK_INTERFACE).address
```

Comment:
@carbonin yes `NETWORK_INTERFACE` is coming from appliance_console. That should be reworked but I think that is beyond the scope of this PR. Also I am only using it as the default and allow the user to provide a different value.